src/main/kotlin/no/nav/medlemskap/regler/common/Personfakta.kt

```diff
@@ -107,7 +107,7 @@ class Personfakta(private val datagrunnlag: Datagrunnlag) {
     fun hentBrukerinputArbeidUtenforNorge(): Boolean = datagrunnlag.brukerinput.arbeidUtenforNorge
 
     fun hentBrukerLandskodeInnenfor12Mnd(): List<Adresse> {
-        return datagrunnlag.personhistorikk.bostedsadresser.filter { it.landkode == "NOR" && it.fom!!.isAfter(datohjelper.kontrollPeriodeForNorskAdresse().fom) }
+        return datagrunnlag.personhistorikk.bostedsadresser.filter { it.landkode == "NOR" && it.tom?.isAfter(datohjelper.kontrollPeriodeForNorskAdresse().fom) ?: it.tom == null }
```

Comment:
Kan gi falsk positiv dersom statsborgerskapet i Norge starter etter kontrollperiodeForNorskAdresse
Kan sette true etter elvis-operator, for mer tydelig kode